api/proto/disperser/v2/disperser_v2.proto

```diff
@@ -1,18 +1,28 @@
 syntax = "proto3";
 package disperser.v2;
+
 import "common/common.proto";
 import "common/v2/common_v2.proto";
+
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2";
 
 // Disperser defines the public APIs for dispersing blobs.
+//
+// buf:lint:ignore SERVICE_SUFFIX // Skip this rule to not create a breaking change.
```

Comment:
here and below: why do we skip so many rules?

Reply:
Because I didn't want to create breaking changes. Otherwise the names of the packages/services/requests/responses would have to change to meet the proto code guidelines, but that would mean any people using those would now have to deal with the breaking change.

3 options here, which do you prefer?:
1. we leave as is here skipping any previous names that don't follow standard. We'll only follow the linter for future packages/services/requests/responses
2. we make the breaking changes
3. we keep these old names, mark them as deprecated, and create new ones that follow the convention

Reply:
The only thing we can do afaiu is ignore entire files from being linted, by adding them to `buf.yaml`.
I opted against that option in case we decide to add new messages/services to the current files, since then they would also not be linted.
What we could do though is to ignore all of our current files, and make sure any new additions are added to new files

Reply:
Ah looks like this comes just a bit late, as it may catch the v2 proto changes.
I personally prefer not to be too intrusive on existing protos like adding a lot ignore comments (they are visible to readers as we generate docs), unless we have strong reasons to do so.

Reply:
Ok removed all of them and added to `ignore_only` section in buf.yaml